cmd/vault-k8s-secret/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+config_auth () {
+  if [ -z "${GOOGLE_APPLICATION_CREDENTIALS}" ]; then
+    echo "PLUGIN_GOOGLE_APPLICATION_CREDENTIALS environment variable is not set." > /dev/null 2>&1
+    exit 1
+  else
+    gcloud auth activate-service-account --key-file "$GOOGLE_APPLICATION_CREDENTIALS" && \
+    gcloud auth list && \
+    gcloud config set account ${FUNCTION_IDENTITY} && \
+    gcloud container clusters get-credentials ${CLUSTER_NAME} --zone ${GCLOUD_ZONE}
```

Comment:
☝️ How does this work for regional k8s clusters that don't have a specific zone?

Reply:
Doesn't. Was just enough of a p.o.c. script to get the general concept working quickly, I pulled that from an old repo of mine and just modified a few things. Not even sure it's a good way to be auth'ing since it's inside the cluster already but wasn't sure how to use k8s auth roles confidently either

Reply:
Roger dodger!  👍